chore: BROS-16: Experiments with tests [WIP]

Gondragos opened the pull request:
<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

Gondragos committed:
Try zero waitForAction timeout in tests

Gondragos committed:
test: add integration test for DateTime control

It's a copy of the same test in e2e tests

Gondragos committed:
fix: massive tests fix

Replaced most prolonged `I.wait` calls with shorter and consistent wait durations to improve test performance.
- Adjusted delays to `0.016s` or other minimal values where applicable.
- Removed unused comments and disabled scenarios where appropriate.
- Enhanced test stability and responsiveness while maintaining accuracy.

niklub commented on web/libs/editor/tests/e2e/codecept.conf.js:
```diff
@@ -22,7 +22,7 @@ module.exports.config = {
       restart: "context",
       timeout: 60000, // Action timeout after 60 seconds
-      waitForAction: headless ? 300 : 1200,
+      waitForAction: 0,
```
return headless mode

Gondragos committed:
test: temporary focus date-time tests for debugging

Enabled `.only` method to isolate and debug the date-time test scenarios. This ensures these tests are executed exclusively for troubleshooting specific issues.

Gondragos committed:
fix: update Cypress config for targeted test specs

Updated the Cypress configuration to include a specific specPattern for integration tests targeting files with ".only.cy" suffix. This change ensures only specific test files are executed during e2e testing. Renamed a test file to align with the updated specPattern.

Gondragos committed:
test: add integration test for transformer functionality

Gondragos committed:
test: switch to temporary empty tests

Added temporary test cases for Cypress and CodeceptJS to evaluate specific metrics during test running.

robot-ci-heartex commented:
This PR is stale because it has been open 45 days with no activity. Remove `stale` label or comment or this will be closed in 10 days.

robot-ci-heartex commented:
This PR was closed because it has been stalled for 10 days with no activity.
